docs/getting_started/blinky_cli.md

```diff
@@ -71,6 +71,17 @@ For example, for the board K64F and the ARM Compiler 5:
 mbed compile -m K64F -t ARM
 ```
 
+If you don't know the name of your target board, there are several ways to tell.
+
+* Invoke ``mbed detect`` and the mbed CLI tool will display an output similar to below, where 'K64F' will be the name of your target platform, and COM3 will be the name of the serial port that the platform is connected to:
+```
+[mbed] Detected "K64F" connected to "D:" and using com port "COM3"
+```
+
+* Check the board information page on the list of [mbed enabled boards](https://developer.mbed.org/platforms/). On the right-hand side of each information page, the name of the target is listed.
+
+* If you only have one mbed enabled board connected, mbed CLI can automatically detect the target by specifing ``-m detect``.
```

Comment:
This looks like a partial command. Does -m detect work for compile and test verbs?

Reply:
@sg -m detect is a valid argument for mbed compile, for example here is the output using mbed compile -m detect:

```
mbed Workspace\mbed-os-example-blinky> mbed compile -m detect -t GCC_ARM
[mbed] Detected "K64F" connected to "D:" and using com port "COM3"
Building project mbed-os-example-blinky (K64F, GCC_ARM)
Scan: .
Scan: env
Scan: mbed
Scan: FEATURE_LWIP
Scan: FEATURE_STORAGE
+------------------+-------+-------+------+
| Module           | .text | .data | .bss |
+------------------+-------+-------+------+
| [fill]           |    86 |     4 | 2369 |
| [lib]/libc.a     | 22929 |  2472 |   89 |
| [lib]/libgcc.a   |  3120 |     0 |    0 |
| [lib]/libnosys.a |    32 |     0 |    0 |
| [lib]/misc       |   248 |     8 |   28 |
| main.o           |    56 |     0 |    4 |
| mbed-os/features |    42 |     0 |  184 |
| mbed-os/hal      |   532 |     0 |   24 |
| mbed-os/platform |  1236 |     4 |  270 |
| mbed-os/rtos     |  8481 |   180 | 5728 |
| mbed-os/targets  |  8007 |    12 |  384 |
| Subtotals        | 44769 |  2680 | 9080 |
+------------------+-------+-------+------+
Total Static RAM memory (data + bss): 11760 bytes
Total Flash memory (text + data): 47449 bytes
```